src/Symfony/Component/Serializer/Normalizer/AbstractNormalizer.php

```diff
@@ -118,6 +118,11 @@ abstract class AbstractNormalizer implements NormalizerInterface, DenormalizerIn
      */
     public const REQUIRE_ALL_PROPERTIES = 'require_all_properties';
 
+    /**
+     * Use class name as default expected type when throwing NotNormalizableValueException instead of unknown.
+     */
+    public const USE_CLASS_AS_DEFAULT_EXPECTED_TYPE = 'use_class_as_default_expected_type';
```

Comment:
IMHO, having a more precise exception message can be kind of considered as a fix (see https://github.com/symfony/symfony/pull/53108), so I think we can forget that context attribute